ambry-store/src/main/java/com.github.ambry.store/DiskIOScheduler.java

```diff
@@ -60,13 +60,20 @@ long getSlice(String jobType, String jobId, long usedSinceLastCall) {
   }
 
   /**
+   * Disables the DiskIOScheduler i.e. there will be no more blocking calls
+   */
+  void disable() {
+    for (Throttler throttler : throttlers.values()) {
+      throttler.disable();
+    }
+  }
+
+  /**
    * Release resources and close throttlers
    */
   @Override
   public void close() {
-    for (Throttler throttler : throttlers.values()) {
```

Comment:
why moved this code to disable? If we don't plan to use close(), why DiskIOScheduler needs to implement closeable ?